docs/plugins/integrations.asciidoc

```diff
@@ -0,0 +1,18 @@
+[[plugin-integrations]]
+== Plugin integrations
+
+Plugin integrations combine related plugins--inputs, outputs, and sometimes
+filters and codecs--into one package. 
+
+|=======================================================================
+| Plugin Integration | Description | Github repository
+| <<plugins-integration-kafka,kafka>> | Plugins for use with the Kafka distributed streaming platform. | https://github.com/logstash-plugins/logstash-integration-kafka[logstash-integration-kafka]
+| <<plugins-integration-rabbitmq,rabbitmq>> | Plugins for processing events to or from a RabbitMQ exchange. | https://github.com/logstash-plugins/logstash-integration-rabbitmq[logstash-integration-rabbitmq]
```

Comment:
If I recall correctly, in RabbitMQ, one reads from queues and sends to exchanges. More on the AMQP 0.9.1 model here: https://www.rabbitmq.com/tutorials/amqp-concepts.html#amqp-model

So maybe "Plugins for processing events to or from a RabbitMQ broker"?